modules/cnf-troubleshooting-common-ptp-operator-issues.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-troubleshooting-common-ptp-operator-issues_{context}"]
+= Troubleshooting common PTP Operator issues
+
+Troubleshoot common problems with the PTP Operator by performing the following steps.
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the PTP Operator on a bare-metal cluster with hosts that support PTP.
+
+.Procedure
+
+. Check the operator and operands are successfully deployed in the cluster.
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ptp
+----
++
+.Example output
+[source,terminal]
+----
+NAME                            READY   STATUS    RESTARTS   AGE
+linuxptp-daemon-5j265           3/3     Running   0          24h
+linuxptp-daemon-s72q2           3/3     Running   0          24h
+ptp-operator-6b8dcbf7f4-czff8   1/1     Running   0          10d
+----
++
+[NOTE]
+====
+When the event bus is enabled, the number of ready linuxptp-daemon pods is `3/3`. If the event bus is not enabled, `2/2` is displayed.
```

Comment:
```suggestion
When the event bus is enabled, the number of ready `linuxptp-daemon` pods is `3/3`. If the event bus is not enabled, `2/2` is displayed.
```

If I'm interpreting this object type correctly, it needs to be enclosed in [backticks](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).